src/components/SingleList.jsx

```diff
@@ -1,13 +1,36 @@
 import './SingleList.css';
+import { MdOutlineDeleteForever } from 'react-icons/md';
 
-export function SingleList({ name, path, setListPath }) {
-	function handleClick() {
+export function SingleList({
+	name,
+	path,
+	setListPath,
+	deleteList,
+	email,
+	selected,
+}) {
+	// Function to handle selecting a list
+	const handleClick = () => {
 		setListPath(path);
-	}
+	};
+
+	// Function to handle deleting a list
+	const handleDelete = async () => {
+		const listPath = '/' + path;
+		await deleteList(listPath, email);
+		localStorage.setItem('tcl-shopping-list-path', '');
+		window.location.reload();
+	};
 
 	return (
-		<li className="SingleList">
-			<button onClick={handleClick}>{name}</button>
-		</li>
+		<div
+			className={`flex items-center justify-between ${selected ? 'selected' : ''}`}
+		>
+			<li className="SingleList">
+				<button onClick={handleClick}>{name}</button>
+			</li>
+			<hr />
+			<MdOutlineDeleteForever className="text-red-700" onClick={handleDelete} />
```

Comment:
The delete button should be commented out as well

Reply:
check again working now